build.gradle.kts

```diff
@@ -19,27 +19,33 @@ repositories {
 dependencies {
     implementation("org.virtuslab:pulumi-kotlin:0.9.4.0")
 
-    implementation("com.squareup:kotlinpoet:1.14.2")
-    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.5.1")
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8:1.7.2")
+    implementation("com.squareup:kotlinpoet:1.16.0")
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.6.3")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8:1.8.0")
 
-    implementation("com.github.ajalt.clikt:clikt:3.5.4")
+    implementation("com.github.ajalt.clikt:clikt:4.3.0")
 
-    implementation("com.squareup.tools.build:maven-archeologist:0.0.10")
+    // https://nvd.nist.gov/vuln/detail/CVE-2023-3635
```

Comment:
What is this comment about?

Reply:
It's about the exclusions below. The maven-archeologist dependency was getting highlighted by IntelliJ due to a transitive vulnerability, so I replaced the problematic depenencies (`com.squareup`). I can make the comment more verbose

Reply:
Done, I added a comment